opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -692,15 +695,35 @@ void Schedule::iterateScheduleSection(std::size_t load_start, std::size_t load_e
     }
 
     void Schedule::shut_well(const std::string& well_name, std::size_t report_step) {
-        this->updateWellStatus(well_name, report_step, Well::Status::SHUT);
+        this->internalWELOPENACTIONXFromPYACTION(well_name, report_step, "SHUT");
     }
 
     void Schedule::open_well(const std::string& well_name, std::size_t report_step) {
-        this->updateWellStatus(well_name, report_step, Well::Status::OPEN);
+        this->internalWELOPENACTIONXFromPYACTION(well_name, report_step, "OPEN");
     }
 
     void Schedule::stop_well(const std::string& well_name, std::size_t report_step) {
-        this->updateWellStatus(well_name, report_step, Well::Status::STOP);
+        this->internalWELOPENACTIONXFromPYACTION(well_name, report_step, "STOP");
+    }
+
+    void Schedule::internalWELOPENACTIONXFromPYACTION(const std::string& well_name, std::size_t report_step, const std::string& wellStatus) {
```

Comment:
Yes, I'd prefer WELOPEN over WELLSTATUS because the keyword I use here is called WELOPEN. What's your opinion on that?
I'm very happy about other suggestions  though!